Portable.Ninject.nuspec

```diff
@@ -19,6 +19,22 @@
     <summary>IoC container for .NET</summary>
     <releaseNotes>Xamarin support for iOS and Android</releaseNotes>
     <tags>Ninject ioc di portable pcl xamarin ios android monotouch monoandroid</tags>
+    <dependencies>
+      <group targetFramework="dotnet">
```

Comment:
I would recommend using `NuSpec.ReferenceGenerator` to add/update the dependencies as they can change over time and the tool will ensure they're up-to-date.